app/workflow_manager/models/analysis_context.py

```diff
@@ -10,8 +10,12 @@ class AnalysisContextStatus(models.TextChoices):
 
 
 class AnalysisContextUseCase(models.TextChoices):
-    COMPUTE = "COMPUTE"
-    STORAGE = "STORAGE"
+    # State of validation: UNKNOWN, PENDING, VERIFIED, VALIDATED, REJECTED
+    VALIDATION = "VALIDATION"
+    # Approval use case: ctTSO, WGS, WGTS,...
+    # A VALIDATION state of VALIDATED is required for any APPROVAL use case
+    # E.g. only fully validated analysis can be used for accredited workloads
+    APPROVAL = "APPROVAL"
```

Comment:
Not sure I understand the 'APPROVAL' field, or is this a 'list of approved use cases' for this analysis? In that case, shouldn't this instead be a list?